src/swap/SwapScreen.test.tsx

```diff
@@ -278,7 +278,7 @@ describe('SwapScreen', () => {
     })
 
     mockExperimentParams.mockReturnValue({
-      swappingNonNativeTokensEnabled: false,
+      swapBuyAmountEnabled: true,
     })
```

Comment:
nit: could we even remove this?

Reply:
Yes, there is another [task](https://linear.app/valora/issue/RET-920/remove-swap-buy-amount-experiment) to remove this. So we do it in separate PRs

Reply:
Ok I got confused as I didn't see that value in the existing test.

Reply:
There was an error in a test if I removed the `mockExperimentParams.mockReturnValue` completely.
I'll handle that in the other task.